portal/pages/status/index.tsx

```diff
@@ -27,7 +19,10 @@ const Status = () => {
       url: `${BUGOUT_STATUS_URL}/status`,
     });
     const list = response.data.map((s: any) => {
-      const name = serviceNames.get(s.name) ?? s.name.replace(/_/g, " ").replace(/api/g, "API");
+      let name = s.normalized_name;
+      if (name === "") {
```

Comment:
I don't know if normalized_name is guaranteed to be present so I think it's better to use if (!name) here. It covers both empty string and absence of value